.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
Add a new line at the bottom

Reply:
Man that's markup and css, there is no need for a newline at the bottom. Read the docs well